spec/minimax_spec.rb

```diff
@@ -12,8 +12,8 @@
     @game = Game.new(@board, @player_1, @player_2)
   end
 
-  describe '#best_move' do
-    it 'returns the best move when there are two moves left (example 1)' do
+  describe 'Human plays first' do
```

Comment:
Personally I would use `describe '#best_move'` and `context 'human plays first'`.

Reply:
commit 18ccf168d1ae744142f8d8f7435a35f67af87b2d